src/app/beer_garden/db/mongo/models.py

```diff
@@ -474,12 +480,16 @@ def clean(self):
                 f"consistent with has_parent value of {self.has_parent}"
             )
 
+        if "status" in self.changed_fields:
+            self.status_updated_at = datetime.datetime.utcnow()
+
     def clean_update(self):
         """Ensure that the update would not result in an illegal status transition"""
         # Get the original status
         old_status = Request.objects.get(id=self.id).status
 
         if self.status != old_status:
+            self.status_updated_at = datetime.datetime.now()
```

Comment:
Sorry I missed this in the last couple rounds of review, but this should be `utcnow()` instead of `now()`.

I think last time I checked I convinced myself that this `clean_update` method will never get called, but it's still probably better to make the fix.